recipes/holopy/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "HoloPy" %}
+{% set version = "3.0.0" %}
+{% set sha256 = "fecfcdd3fb46de5b03db7ee53fa9ef6463eb75a1889143578b51707ecaeb878e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://github.com/manoharan-lab/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k or win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - numpy x.x
+    - setuptools
+    - gcc
+    - toolchain3
+
+  run:
+    - python
+    - numpy x.x
+    - scipy
+    - pyyaml
+    - pillow
+    - h5py
+    - emcee
+    - matplotlib
+    - xarray >=0.9
+    - h5netcdf
+    - libgcc  # [linux]
+    - libgfortran  # [osx]
+    
+test:
+  imports:
+    - holopy
+  requires:
+    - nose
+
+about:
+  home: http://manoharan.seas.harvard.edu/holopy
+  license: GPL v3
+  license_file: COPYING
+  summary: 'Holopy is a python based tool for working with digital holograms and light scattering'
+
+  description: |
+    HoloPy is a software package for computational light scattering and digital holography. 
+    It has tools for managing data and metadata, doing reconstructions, computing light scattering
+    from micron scale particles or clusters of such particles, and fitting scattering models to
+    digital holograms to measure particle parameters. HoloPy is free and open source software (GPL v3).
+  doc_url: http://holopy.readthedocs.io/
+  dev_url: https://github.com/manoharan-lab/holopy
+
+extra:
+  recipe-maintainers:
+    - barkls
+    - vnmanoharan
```

Comment:
I think we confirmed this before (maybe in the old PR), but just to check. Are you ok being a maintainer here, @vnmanoharan?